docusaurus/docs/migration-guides/angular-react/convert-value-and-range-maps.md

```diff
@@ -0,0 +1,88 @@
+﻿---
+id: angular-react-convert-mappings
+title: Convert value and range maps
+sidebar_position: 3
+description: How to migrate an Angular plugin that uses value and range maps to React
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - React
+  - ReactJS
+  - Angular
+  - migration
+  - rangemap
+  - valuemap
+---
+
+# Convert value and range maps
+
+Grafana plugins built using the AngularJS SDK could use editors for value and range maps. When updating your plugin to React, it is necessary to convert the configuration options of these maps to a new format.

```

Comment:
I found the draft's opening paragraph a bit unclear. I've tried to make it more clear, but be sure that I haven't erred. Also, I would like to find the exact spelling of "value and range maps" in the AngularJS documentation, but I could find it. Do you have a link for me to look at?

Reply:
this is what i've located https://grafana.com/docs/grafana/latest/panels-visualizations/configure-value-mappings/

Reply:
much better - looks like i didn't proof read that enough.